src/with-unique.js

```diff
@@ -0,0 +1,14 @@
+import { HTMLElement, sym, uniqueId } from './util';
+
+const _is = sym();
+
+export function withUnique (Base = HTMLElement) {
+  return class extends Base {
+    static get is () {
+      return this[_is] || (this[_is] = `x-${uniqueId().substring(0, 8)}`);
```

Comment:
just a nitpick, but there is "too much going on" in that getter.

Maybe extracting ``x-${uniqueId().substring(0, 8)}`` to `generateUniqueName()` function would be more readable 

```js
// util.js

export function generateUniqueName(prefix='x'){
  return `${prefix}-${uniqueId().substring(0, 8)}`
}


// with-unique.js

static get is () {
  return this[_is] || (this[_is] = getUniqueName())
}

```

WDYT?